app/src/main/java/com/keylesspalace/tusky/TuskyApplication.kt

```diff
@@ -38,11 +42,21 @@ class TuskyApplication : Application(), HasAndroidInjector {
 
     @Inject
     lateinit var androidInjector: DispatchingAndroidInjector<Any>
+
     @Inject
-    lateinit var notificationWorkerFactory: NotificationWorkerFactory
+    lateinit var notificationWorkerFactory: Lazy<NotificationWorkerFactory>
 
     override fun onCreate() {
-
+        // Uncomment me to get StrictMode violation logs
```

Comment:
How about enabling it for debug builds?

Reply:
we get spam because of the SharedPreferences/Conscrypt so I don't know

Reply:
I wouldn't mind in debug builds, but you can also leave it commented